packages/applications-service-api/src/services/advice.service.js

```diff
@@ -47,34 +53,24 @@ const getAllAdvice = async (query) => {
 	};
 };
 
-const getAdviceById = async (adviceID) => {
-	const advice = await db.Advice.findOne({
-		where: {
-			adviceID
-		}
-	});
-
-	if (!advice) return undefined;
-
-	const attachments = await db.Attachment.findAllAttachmentsWithCase(advice.caseReference, {
-		where: {
-			adviceID
-		}
-	});
-
-	const adviceDTO = advice.get({
-		plain: true
-	});
-
-	adviceDTO.attachments = attachments.map((attachment) => {
-		// eslint-disable-next-line no-unused-vars
-		const { adviceID, ...dto } = attachment.get({
-			plain: true
+const getAdviceById = async (adviceID, caseReference) => {
+	if (isBackOfficeCaseReference(caseReference)) {
+		const advice = await getBackOfficeAdviceById(adviceID);
+		if (!advice) return undefined;
```

Comment:
nitpick for redundant `undefined`
```suggestion
		if (!advice) return;
```